src/bp-messages/classes/class-bp-messages-notices-admin.php

```diff
@@ -58,7 +56,7 @@ public static function register_notices_admin() {
 			$bp->messages->admin = new self;
 		}
 
-		return $bp->messages->admin;
+		$bp->messages->admin;
```

Comment:
Same thing here, I'm a bit nervous about this change. I need to be sure it doesn't break the notices management screen.

Reply:
I tested this and didn't see any issues or warnings.

Reply:
Just tested this. I saw no side effects. It's could to go.